docs/concepts/workloads/pods/disruptions.md

```diff
@@ -0,0 +1,260 @@
+| pod-d | not-available | node-2 | normal     |
+
+At this point, if an impatient cluster administrator tries to drain `node-2` or
+`node-3`, they will not succeed, because there are only 2 available pods for the deployment, and its PDB requires at least 2.
+
+After some time passes, `pod-d` becomes available, and, independently, `pod-a` finishes terminating.
+
+(Note: for a StatefulSet, the pod would need to terminate completely before the controller
+would create a replacement, but otherwise, things work the same.)
+
+The cluster state now looks like this:
+
+| pod   | pod status  | node   |  node status |
+|:-----:|:-----------:|:------:|:------------:|
+|       |             | node-1 | drained      |
+| pod-b |  available  | node-2 | normal       |
+| pod-c |  available  | node-3 | normal       |
+| pod-d |  available  | node-2 | normal       |
+
+Now, the cluster admin tries again to drain `pod-b`.
+The drain command will try, to evict `pod-b` and `pod-d`.  One will succeed
+but the second will fail, and it will retry until a replacement, `pod-e`,
+becomes available.  If there are not enough resources in the cluster to schedule
+`pod-e` for it to start, then the drain will block.  The cluster may end up in this
+state:
+
+| pod   | pod status  | node   |  node status |
+|:-----:|:-----------:|:------:|:------------:|
+|       |             | node-1 | drained      |
+| pod-b |  *deleted*  | node-2 | draining     |
+| pod-c |  available  | node-3 | normal       |
+| pod-d |  available  | node-2 | draining     |
+| pod-e |  available  | node-3 | normal     |
+| pod-f |  unscheduled  |      |           |
+
+At this point, the cluster administrator needs to
+add a node back to the cluster to proceed with the upgrade.
+
+You can see how Kubernetes determine the rate at which disruptions, like node upgrades,
+can happen, varying the rate as the operation progresses, according to:
+
+- how many replicas an application needs
+- how long it takes to gracefully shutdown an instance
+- how long it takes a new instace to start up
+- the type of controller
+- the cluster's capacity for new pods
+
+## Separating Cluster Owner and Application Owner Roles
+
+Often, it is useful to think of the Cluster Manager
+and Application Owner as separate roles with limited knowlege
+of each other.   This separation of responsibilities
+may make sense when:
+
+- when there are many application teams sharing a Kubernetes cluster, and 
+  there is natural specialization of roles
+- when third-party tools or services are used to automate cluster management
+
+Pod Disrutption Budgets supports this separation of roles by providing an
+interface between the roles.
+
+If you do not have such a separation of responsibilities in your organization,
+you may not need to use Pod Disruption Budgets.  In that case, when performing
+cluster management tasks, like upgrading node software, and so on, you will
+think the impact on your application at each step.
```

Comment:
Whether consider or think, I believe we're missing something like a "need to" after "will".